lib/config.js

```diff
@@ -390,7 +390,8 @@ AWS.Config = AWS.util.inherit({
     convertResponseTypes: true,
```

Comment:
Would you mind adding some docs around this new config parameter?
Something like this at line 98:
```
 * @!attribute signatureCache
 *   @return [Boolean] whether the signature to sign requests with (overriding
 *     the API configuration) is cached. Only applies to the signature version 'v4'.
 *     Defaults to `true`.
```
and something like this at line 185:
``` 
   * @option options signatureCache [Boolean] whether the signature to sign
   *   requests with (overriding the API configuration) is cached. Only applies
   *   to the signature version 'v4'. Defaults to `true`.
```